plugins/modules/glue_crawler.py

```diff
@@ -31,6 +31,12 @@
       - The name you assign to this crawler definition. It must be unique in your account.
     required: true
     type: str
+  purge_tags:
```

Comment:
@ichekaldin purge_tags and tags are already documented by extending doc fragments with `amazon.aws.tags ` as you can see here https://github.com/ansible-collections/community.aws/pull/1399/files#diff-6e966a9091558c18c1b64c4a6a425d5240c385b3f3f4279c59bee37ad55463d7R91-R94

This is the content of the tags doc fragment https://github.com/ansible-collections/amazon.aws/blob/main/plugins/doc_fragments/tags.py

Reply:
@alinabuzachis I see. I didn't realize this was the case. Thank you, I'll withdraw this PR.